content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -37,13 +37,18 @@ these certificates will validate against the cluster root CA.
 
 ## Trusting TLS in a Cluster
 
-Trusting the custom CA from an application running as a pod usually requires
+Trusting the [custom CA](#a-note-to-cluster-administrators) from an application running as a pod usually requires
```

Comment:
The note being linked to here talk about enabling the default implementation of a signer, can you please help me understand how that links to custom/dedicated CA?

Reply:
As I understand it, `--cluster-signing-cert-file` and -`-cluster-signing-key-file` are the options required to specify a custom/dedicated CA.